doc/source/howtos/garbd_howto.rst

```diff
@@ -1,60 +1,68 @@
 .. _garbd_howto:
 
-============================
+=========================================
 Setting up Galera Arbitrator
-============================
+=========================================
 
-`Galera Arbitrator
-<http://galeracluster.com/documentation-webpages/arbitrator.html>`
-is a member of |Percona XtraDB Cluster| that is used for voting
-in case you have a small number of servers (usually two)
-and don't want to add any more resources.
-Galera Arbitrator does not need a dedicated server.
-It can be installed on a machine running some other application.
-Just make sure it has good network connectivity.
+The size of a cluster increases when a node joins the cluster and decreases when a node leaves. The nodes how a cluster reacts to replication problems with inconsistency voting. The size of the cluster determines the required votes to achieve a quorum. If a node no longer responds and is disconnected from the cluster the remaining nodes vote. The majority of the nodes that vote are considered to be in the cluster.
```

Comment:
The nodes how a cluster reacts to replication problems with inconsistency voting. - This sentence seems to be incomplete.

Reply:
Fixed.